tests/functional/spec/features/show_section_summary_on_completion/show_section_summary_on_completion.spec.js

```diff
@@ -1,58 +1,56 @@
+    it("When I reach the end of that section, then I go straight to the hub", () => {
+      $(employmentStatusBlockPage.workingAsAnEmployee()).click();
+      $(employmentStatusBlockPage.submit()).click();
 
-            expect(browser.getUrl()).to.contain(hubPage.url());
-        });
+      expect(browser.getUrl()).to.contain(hubPage.url());
     });
+  });
 
-    describe('Given I have completed a section with the summary turned off for the forward journey', () => {
-        it('When I return to a completed section from the hub, then I am returned to that section summary' , () => {
-            $(hubPage.summaryRowLink(1)).click();
+  describe("Given I have completed a section with the summary turned off for the forward journey", () => {
+    it("When I return to a completed section from the hub, then I am returned to that section summary", () => {
+      $(hubPage.summaryRowLink(1)).click();
 
-            expect(browser.getUrl()).to.contain(employmentSectionSummary.url());
-        });
+      expect(browser.getUrl()).to.contain(employmentSectionSummary.url());
     });
+  });
 
-    describe('Given I am completing a section with the summary turned on for the forward journey', () => {
-        before("Get to hub", () => {
-            browser.url(hubPage.url());
-        });
+  describe("Given I am completing a section with the summary turned on for the forward journey", () => {
+    before("Get to hub", () => {
+      browser.url(hubPage.url());
+    });
 
-        it('When I reach the end of that section, ' +
-            'then I will be taken to the section summary to enable me to amend an answer' , () => {
-            $(hubPage.summaryRowLink(2)).click();
-            $(proxyQuestionPage.noIMAnsweringForMyself()).click();
-            $(proxyQuestionPage.submit()).click();
+    it("When I reach the end of that section, " + "then I will be taken to the section summary to enable me to amend an answer", () => {
+      $(hubPage.summaryRowLink(2)).click();
+      $(proxyQuestionPage.noIMAnsweringForMyself()).click();
+      $(proxyQuestionPage.submit()).click();
 
-            expect(browser.getUrl()).to.contain(accommodationSectionSummary.url());
-        });
+      expect(browser.getUrl()).to.contain(accommodationSectionSummary.url());
     });
+  });
 
-    describe('Given I have completed a section with the summary turned on for the forward journey', () => {
-        before("Get to hub", () => {
-            browser.url(hubPage.url());
-        });
+  describe("Given I have completed a section with the summary turned on for the forward journey", () => {
+    before("Get to hub", () => {
+      browser.url(hubPage.url());
+    });
 
-        it('When I return to a completed section from the hub, then I am returned to the correct section summary' , () => {
-            $(hubPage.summaryRowLink(2)).click();
+    it("When I return to a completed section from the hub, then I am returned to the correct section summary", () => {
```

Comment:
Probably a bit nit picky but I've noticed most files have a capital `T` on `then` after a comma but theres a few files with a lower case `t`. I don't know if you want to standardise this

Reply:
Not sure, I think it should be capitalised for nicer output, but we don't seem to follow that pattern consistently. But I don't think I should change that as part of this PR without a way of enforcing it also.

Reply:
Yeah no worries, just something I noticed